src/apps/app_search.erl

```diff
@@ -41,6 +41,22 @@ add_entry(undefined, _, _, _) -> do_nothing;
 add_entry(PID, Name, Value, ID) ->
 	PID ! {add_tx, Name, Value, ID}.
 
+get_tags_by_id(TXID) ->
+	get_tags_by_id(whereis(http_search_node), TXID).
+get_tags_by_id(PID, TXID) ->
```

Comment:
I think we agreed on, but I cannot find it in the code standard, that functions should have an empty line in between them even if they have the same name.

Reply:
✅